src/systems/systems/src/navigation/radio_altimeter.rs

```diff
@@ -0,0 +1,864 @@
+        let b: Length = self.receiver_height_over_ground();
+
+        // Perform some 2D geometry to determine the shortest path between the two transceivers and
+        // the ground. The basic idea is that, given two transceivers A, B and a line g representing
+        // the ground. We now would like to determine the length of the shortest path from A to B
+        // via a point on the line. To do this, you can can reflect one of the transceivers B
+        // along the ground g to construct B'. The intersection of the line A-B' with g will result
+        // in the reflection point C in g, and this line A-C-B' now is equal to the shortest path
+        // A-C-B. By then calculating some right angles between A, B', and g, we eventually have a
+        // simple pythagorean triangle where the hypotenuse corresponds to A-C-B', which we can
+        // calculate trivially.
+        let transceiver_along_ground_distance = self.antenna_along_ground_distance();
+        let path_in_air: Length = ((a + b) * (a + b)
+            + transceiver_along_ground_distance * transceiver_along_ground_distance)
+            .sqrt();
+        let shortest_path_length: Length =
+            path_in_air + self.transmitter.electric_length + self.receiver.electric_length;
+
+        // At this point we've determined the length of the shortest path between the two
+        // transceivers. We now convert it into a travel time.
+
+        // It would be nice if this could be a const, but unfortunately the Velocity constructor is
+        // too complicated for what Rust deems acceptable for a const.
+        let speed_of_light: Velocity = Velocity::new::<meter_per_second>(299_792_458.);
+
+        // While radio waves propagate at the speed of light in vacuum, they travel slower in other
+        // mediums. This correction is called refractive index.
+        // If you want, you can try calculating yourself using
+        // https://www.fig.net/resources/proceedings/fig_proceedings/fig_2002/Js28/JS28_rueger.pdf
+        // However as the index varies by only around 0.01%, the effects on the final radio height
+        // as measured by civil aircraft is negligible.
+        let refractive_radio_index = Ratio::new::<ratio>(1.000293);
+        let speed_of_radio_waves = speed_of_light / refractive_radio_index;
+
+        let travel_time = shortest_path_length / speed_of_radio_waves;
+
+        if shortest_path_length < Length::new::<foot>(10200.) {
+            Some(TransceiverPairResponse::new(travel_time))
+        } else {
+            None
+        }
+    }
+}
+
+impl SimulationElement for Ala52BTransceiverPair {
+    fn read(&mut self, reader: &mut SimulatorReader) {
+        self.alt_above_ground = Length::new::<foot>(reader.read(&self.alt_above_ground_id));
+        self.pitch = Angle::new::<degree>(reader.read(&self.pitch_id));
+        self.bank = Angle::new::<degree>(reader.read(&self.bank_id));
```

Comment:
**thought**: These are equal to the default unit declared in `src/systems/systems/src/simulation/mod.rs`:
```rust
read_write_uom!(Length, foot);
read_write_uom!(Angle, degree);
```
**suggestion**: Remove the unit declaration:
```suggestion
        self.alt_above_ground = reader.read(&self.alt_above_ground_id);
        self.pitch = reader.read(&self.pitch_id);
        self.bank = reader.read(&self.bank_id);
```